csrc/quantization/cutlass/scaled_mm_dq_c2x.cu

```diff
@@ -0,0 +1,301 @@
+#include <assert.h>
```

Comment:
IMHO, perhaps quantization/cutlass_w8a8/ as folder name would be more appropriate. we may also need to be able to optimize other operators based on cutlass